docs/observatorium/thanos/README.md

```diff
@@ -0,0 +1,19 @@
+## Thanos Access
+
+[Thanos](https://github.com/thanos-io/thanos) is a set of components that can be composed into a highly available metric system with unlimited storage capacity, which can be added seamlessly on top of existing Prometheus deployments.
+
+An instance of Thanos is available for use in OperateFirst.
+
+Thanos access is open to public and the Query API/Console can be accessed at: <br>
+http://thanos-query-frontend-opf-observatorium.apps.zero.massopen.cloud/
+
+
+In this instance of Thanos, metric data will be stored as following:
+
+| Resolution  | Duration    |
+| ----------- | ----------- |
+| Original    | 7 days      |
+| 5 min       | 30 days     |
+| 1 hour      | Indefinite  |
+
+*Note: the resolution and duration of the stored metric data might change in the future*
```

Comment:
Can you link to the Thanos docs in here? Like what does the "original" resolution mean and so on.. Just some links explaining what the settings mean.